pkg/monitor/azure/nsg/nsg_test.go

```diff
@@ -92,6 +98,29 @@ var (
 	nsgOutbound = armnetwork.SecurityRuleDirectionOutbound
 )
 
+func createOC() api.OpenShiftCluster {
```

Comment:
What's the major difference between this func and `ocFactory` below? They seem very similar. Would it be possible to parameterize/merge the diffs into a single func?